src/manager/evm/manager.rs

```diff
@@ -397,6 +397,27 @@ impl<M: Middleware + Send + Sync + 'static> EthManager for EthSubnetManager<M> {
         let initialized = self.gateway_contract.initialized().call().await?;
         Ok(initialized)
     }
+
+    async fn subnet_bottom_up_checkpoint_period(&self, subnet_id: &SubnetID) -> Result<ChainEpoch> {
+        let address = last_evm_address(subnet_id)?;
```

Comment:
Can you remind me what we are doing with `last_evm_address`? It is gestting the subnet actor that is governing the child subnet?